ui/apps/platform/src/Containers/Clusters/InitBundles/InitBundlesTable.tsx

```diff
@@ -0,0 +1,43 @@
+import React, { ReactElement } from 'react';
+import { Link } from 'react-router-dom';
+import { TableComposable, Tbody, Td, Th, Thead, Tr } from '@patternfly/react-table';
+
+import { ClusterInitBundle } from 'services/ClustersService';
+import { clustersInitBundlesPath } from 'routePaths';
+
+export type InitBundlesTableProps = {
+    initBundles: ClusterInitBundle[];
+};
+
+function InitBundlesTable({ initBundles }: InitBundlesTableProps): ReactElement {
+    return (
+        <TableComposable variant="compact" borders={false} id="InitBundlesTable">
```

Comment:
What is the HTML id for?

(It jumped out to me because those are usually kebab-case.)

Reply:
Good eyes! Probably copy paste from administration events table which has a style rule.